components/eamxx/src/diagnostics/atm_tend.cpp

```diff
@@ -0,0 +1,76 @@
+#include "diagnostics/atm_tend.hpp"
+
+#include <ekat/kokkos/ekat_kokkos_utils.hpp>
+
+#include "share/util/scream_universal_constants.hpp"
+
+namespace scream {
+
+AtmTendDiag::AtmTendDiag(const ekat::Comm &comm,
+                         const ekat::ParameterList &params)
+    : AtmosphereDiagnostic(comm, params) {
+  EKAT_REQUIRE_MSG(params.isParameter("Tend Name"),
```

Comment:
Minor, but here "Tend Name" seems misleading: it is not really the name of the tendency, but the name of the field of which the tendency has to be computed.